### PR TITLE
[css-grid-1][editorial] Rearrange the grid layout algo a little for clarity, making a new top-level algo. #3418

### PR DESCRIPTION
**Pull request by tabatkins**
@fantasai and I did some editorial rearrangement, per #3418.

PR to ask for review from @Loirooriol and @mrego.

### CODE REVIEW

**Comment by fantasai** [comment]
Reminder to xcommit to css-grid-2 before merging. :)

File: css-grid-1/Overview.bs [hunk 1]
```diff
@@ -3773,10 +3773,44 @@ Grid Container Baselines</h3>
...
+	which sizes the [=grid container=],
+	sizes and positions all the [=grid tracks=],
+	and lays out the [=grid items=]
+	which have been [=grid item placement algorithm|placed=] into its [=grid areas=].
+
+	1. First, find the size of the [=grid container=],
+		per [[#intrinsic-sizes]].
+
+		Note: During this phase,
+		cyclic <<percentage>> track sizes are treated as ''grid-template-columns/auto''.
```
Comment (Loirooriol) [inline css-grid-1/Overview.bs:3789]: Maybe include `fit-content(<percentage>)` too? E.g.

```suggestion
		cyclic track sizing functions containing a <<percentage>> are treated as ''grid-template-columns/auto''.
```

File: css-grid-1/Overview.bs [hunk 2]
```diff
@@ -3773,10 +3773,44 @@ Grid Container Baselines</h3>
...
+
+	2. Given the resulting [=grid container=] size,
+		run the [=Grid Sizing Algorithm=]
+		to size the [=grid=].
+
+		Note: During this phase,
+		<<percentage>> track sizes are resolved
```
Comment (Loirooriol) [inline css-grid-1/Overview.bs:3796]:
```suggestion
		<<percentage>>s in track sizes are resolved
```

to include `fit-content(<percentage>)`?

**Review by Loirooriol — APPROVED** [review]

File: css-grid-1/Overview.bs (hunk 1 above)
Comment (fantasai) [reply css-grid-1/Overview.bs:3789]: I think, given it's a note, that might be unnecessarily specific. :)

File: css-grid-1/Overview.bs (hunk 2 above)
Comment (fantasai) [reply css-grid-1/Overview.bs:3796]: Maybe we can use similar wording in both notes...

**Review by mrego — APPROVED** [review]
Wow, it's been a while, but this wording seems pretty clear to me. Thanks for working on this!

File: css-grid-1/Overview.bs (hunk 1 above)
Comment (Loirooriol) [reply css-grid-1/Overview.bs:3789]: OK, yeah, it's fine

**Pull request merged**